src/components/ListItem/ListItem.vue

```diff
@@ -0,0 +1,457 @@
+						<span
+							class="list-item-content__line-one__title"
+							:class="{'bold': bold}">
+							{{ title }}
+						</span>
+						<span
+							v-if="hasDetails && !displayActions"
+							class="list-item-content__line-one__details">
+							{{ details }}
+						</span>
+					</div>
+					<div class="list-item-content__line-two"
+						:class="{'bold': bold}">
+						<span v-if="hasSubtitle" class="list-item-content__line-two__subtitle">
+							<!-- @slot Slot for the second line of the component -->
+							<slot name="subtitle" />
+						</span>
+						<span v-if="!displayActions" class="list-item-content__line-two__counter">
+							<AppNavigationCounter
+								v-if="counterNumber !== 0"
+								:highlighted="counterHighlighted">
+								{{ counterNumber }}
+							</AppNavigationCounter>
+						</span>
+					</div>
+				</div>
+				<div
+					v-show="displayActions"
+					class="list-item-content__actions"
+					@click.prevent.stop="">
+					<Actions
+						ref="actions"
+						menu-align="right"
+						:aria-label="actionsAriaLabel"
+						@update:open="handleActionsUpdateOpen">
+						<!-- @slot Provide the actions for the right side quick menu -->
+						<slot
+							name="actions" />
+					</Actions>
+				</div>
+			</div>
+		</a>
+	</nav-element>
+</template>
+
+<script>
+import Actions from '../Actions'
+import AppNavigationCounter from '../AppNavigationCounter'
+
+export default {
+	name: 'ListItem',
+
+	components: {
+		Actions,
+		AppNavigationCounter,
+	},
+
+	props: {
+		/**
+		 * The details text displayed in the upper right part of the component
+		 */
+		details: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * Title
+		 */
+		title: {
+			type: String,
+			required: true,
+		},
+
+		/**
+		* Pass in `true` if you want the matching behaviour to
+		* be non-inclusive: https://router.vuejs.org/api/#exact
+		*/
+		exact: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		* The route for the router link.
+		*/
+		to: {
+			type: [String, Object],
+			default: '',
+		},
+
+		/**
+		 * Id for the <a> element
+		 */
+		anchorId: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * Boldens title and subtitle
+		 */
+		bold: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Toggle the active state of the component.
+		 */
+		active: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Aria label for the wrapper element.
+		 */
+		linkAriaLabel: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * Aria label for the actions toggle.
+		 */
+		actionsAriaLabel: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * If different from from 0 this component will display the
+		 * AppNavigationCounter component
+		 */
+		 counterNumber: {
+			 type: Number,
+			 default: 0,
+		 },
+
+		/**
+		 * Toggles the highlighted state of the counter
+		 */
+		 counterHighlighted: {
+			 type: Boolean,
+			 default: false,
+		 },
+
+	},
+
+	data() {
+		return {
+			hovered: false,
```

Comment:
`hovered` doesn't seems to be used anywhere else?

Reply:
This is used in the `menuOpen` watcher